plugins/rtmp-services/data/services.json

```diff
@@ -2617,4 +2624,4 @@
             }
         }
     ]
-}
+}
```

Comment:
End files with a newline. If this was automated, we may need to adjust the formatter.